<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmConsultaClienteListagem</class>
 <widget class="QMainWindow" name="FrmConsultaClienteListagem">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista de Clientes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(42, 66, 76);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>102</y>
      <width>736</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView_consulta_cliente">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
        background-color: transparent;
        border-style: solid;
        border-width: 1px;
        border-color: rgb(103, 120, 138);
border-radius:3px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::NoPen</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>7</y>
      <width>851</width>
      <height>59</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="3">
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="total_registros_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:3px</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="cad_cli_00_doc_localiza_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255,255, 255);
background-color: rgb(0, 0, 47);
border-radius:3px</string>
       </property>
       <property name="text">
        <string>     Registros    </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="cad_cli_00_nome_localiza">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
        background-color: transparent;
        border-style: solid;
        border-width: 1px;
        border-color: rgb(103, 120, 138);
border-radius:3px;</string>
       </property>
       <property name="maxLength">
        <number>62</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="cad_cli_00_nome_localiza_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:3px</string>
       </property>
       <property name="text">
        <string>Insira o Nome</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>756</x>
      <y>450</y>
      <width>101</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cli_listagem">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color:rgb(255,255, 255);
background-color: rgb(0, 0, 47);
        border-style: solid;
        border-width: 1px;
        border-color: rgb(103, 120, 138);
border-radius:3px}

QPushButton:hover{
color: rgb(0, 0, 47);
background-color: rgb(192, 206, 255);
        border-style: solid;
        border-width: 1px;
        border-color: rgb(103, 120, 138);
border-radius:3px}

QPushButton:pressed {
    background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5,
        stop: 0 rgb(255, 255, 255));
    border-color: rgb(0, 0, 47);
border-radius:3px}
</string>
       </property>
       <property name="text">
        <string>Confirmar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Users/Usuario/imagens/icon-add.png</normaloff>../../../Users/Usuario/imagens/icon-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cli_sair">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color:rgb(255,255, 255);
background-color: rgb(0, 0, 47);
        border-style: solid;
        border-width: 1px;
        border-color: rgb(103, 120, 138);
border-radius:3px}

QPushButton:hover{
color: rgb(0, 0, 47);
background-color: rgb(192, 206, 255);
        border-style: solid;
        border-width: 1px;
        border-color: rgb(103, 120, 138);
border-radius:3px}

QPushButton:pressed {
    background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5,
        stop: 0 rgb(255, 255, 255));
    border-color: rgb(0, 0, 47);
border-radius:3px}

</string>
       </property>
       <property name="text">
        <string>Sair</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Users/Usuario/imagens/icons-sinal-saída.png</normaloff>../../../Users/Usuario/imagens/icons-sinal-saída.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
